pages/data-migration/knowledge-graph.mdx

```diff
@@ -0,0 +1,571 @@
+
+ Now, passing that information ot the LLM model, we can ask questions like:
+
+"Are we able to execute this project?" 
+
+Here is the response of the LLM model (short version):
+
+```plaintext
+
+Project Overview
+
+- Project Name: Data Preprocessing
+    - Budget: $50,000
+    - Start Date: 2024-10-01
+    - End Date: 2024-11-30
+    - Total Duration: 60 days
+
+Tasks and Requirements
+
+- Task Name: Data Cleaning
+    - Description: Clean and preprocess raw data
+    - Estimated Man-Days: 10 days
+    - Required Skill: Data Analysis (inferred from the relationship)
+
+Personnel Involved
+
+- Person: Jane Smith
+  - Cost per Day: $600
+  - Skills: Data Analysis (inferred from the relationship)
+
+Conclusion
+
+Based on the analysis, the project does have a sufficient level of skilled
+personnel to fulfill the requirements, as Jane Smith is qualified for the
+necessary task. Additionally, the budget is adequate to cover her costs, with a
+significant amount remaining for any additional needs or tasks that may arise.
+```
+
+Now if you take a closer look, the the LLM model has inferred some knowledge
+from the Knowledge graph. It took skills that people have and are requried for
+task, and pe In a sense, without even thinking about sepcifc things you want to
+ask, it is inffering what is there in the knowledge graph. 
+
+Passing in the same data and asking the question: "Do employees work on tasks and 
+projects that match their domain preferences?"
+
+Here is the response of the LLM model:
+
+
+```plaintext
+Yes, it appears that people are working on tasks they are interested in:
+
+Task Interest Alignment: Jane Smith, who is assigned to work on the "Data
+Cleaning" task, has a preference for the Data Science domain, which is related
+to this task. This alignment suggests she is working within an area of interest.
+
+Skills and Preferences: Jane possesses the Data Analysis skill, which is also a
+requirement for the project and a direct need for the "Data Cleaning" task. This
+indicates that her skills and interests align with her assigned
+responsibilities. 
+
+This structure suggests that team members like Jane are indeed
+working on tasks that match both their skills and their domain preferences,
+which can enhance motivation and project outcomes.
+```
+
+## Memgraph's office hours
+
+Schedule a 30 min session with one of our engineers to discuss how Memgraph fits
+with your architecture. Our engineers are highly experienced in helping
+companies of all sizes to integrate and get the most out of Memgraph in their
+projects. Talk to us about data modeling, optimizing queries, defining
+infrastructure requirements or migrating from your existing graph database. No
+nonsense or sales pitch, just tech.
+
+![](/pages/getting-started/memgraph-office-hours.svg)
+
+<Cards>
+  <Card
+    title="Book a call"
+    href="https://memgraph.com/office-hours"
+  />
+</Cards>
+
+
+
+## Dataset for seeding the example
```

Comment:
Make this collapsable and place it somewhere above.

Reply:
I am not sure how to collapse this; I didn't find any solution on the web; what do you say we upload this to the best practices repo under the kg folder?